src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -132,20 +132,20 @@ protected AbstractEntityAIBasic(@NotNull final J job)
           /*
             Init safety checks and transition to IDLE
            */
-          new AITarget(this::initSafetyChecks, true),
+          new AITarget(this::initSafetyChecks, true, 10),
           /*
             Update chestbelt and nametag
             Will be executed every time
             and does not stop execution
            */
-          new AITarget(this::updateVisualState, true),
+          new AITarget(this::updateVisualState, true, 10),
```

Comment:
Might have to check at the guards after that, they run their logic only every 20 ticks, so this might relay this too strongly (we can reduce the delay at the guards for that)